app/assets/src/components/ui/icons/info_icon.scss

```diff
@@ -0,0 +1,22 @@
+// TODO(tiago): find a way to get rid of these styles
+.cls1 {
```

Comment:
hm interesting. you didn't want to just leave them in the svgs?

Reply:
Had syntax error in React. Not sure what is the best way to incorporate it, so ended up with a quick fix. With more time I can turn them into properties, probably.
Let me know if you know of a better way...